pipeline/transformer/transformer_tei/src/main/scala/weco/pipeline/transformer/tei/transformers/TeiPhysicalDescription.scala

```diff
@@ -0,0 +1,136 @@
+    * <msPart xml:id="">
+    *  <physDesc>
+    *    <objectDesc>
+    *      <supportDesc>
+    *        <support>Multiple manuscript parts collected in one volume.</support>
+    *      </supportDesc>
+    *    </objectDesc>
+    */
+  private def physicalDescription(nodeSeq: NodeSeq) =
+    (nodeSeq \ "physDesc" \\ "supportDesc").map { supportDesc =>
+      val materialString = (supportDesc \@ "material").trim
+      val material =
+        if (materialString.nonEmpty) s"Material: $materialString" else ""
+      val support = parseSupport(supportDesc).toList
+      val extent = parseExtent(supportDesc).toList
+      (support ++ List(material) ++ extent).filterNot(_.isEmpty).mkString("; ")
+
+    }.headOption
+
+  /**
+    * The extent contains information about the page count and dimension of the manuscript:
+    * <extent>3 pages
+    *  <dimensions unit="mm" type="leaf">
+    *    <height>100mm</height>
+    *    <width>300mm</width>
+    *  </dimensions>
+    * </extent>
+    */
+  private def parseExtent(supportDesc: Node) = {
+    val extent = supportDesc \ "extent"
+    val extentStr = if (extent.exists(_.child.size > 1)) {
+      val extentLabel = extent
+        .flatMap(_.child)
+        .collect { case node if node.label != "dimensions" => node.text.trim }
+        .mkString(" ")
+        .trim
+      val dimensions = parseDimensions(extent)
+      (extentLabel +: dimensions).filterNot(_.isEmpty).mkString("; ")
+    } else extent.text.trim
+    NormaliseText(extentStr)
+  }
+
+  /**
+    * The support contain information about the material, the description and the watermarks present:
+    * <support>Paper, folded in 2.
+    *   <watermark>First watermark very similar to <ref>Mosin and Traljic 6947, 6949, 6956</ref> (saucisson), attested in 1338-50.</watermark>
+    *   <watermark>Second watermark identical with <ref>Mosin and Traljic 5791</ref> (licorne), attested in 1339-44.</watermark>
+    *   <measure type="chainline">Chain distance 43 mm.</measure>
+    *   Added leaves (folios 1 and 198): paper folded in 2.
+    *   <watermark>Watermarks similar to <ref>Piccard 122415</ref> (scissors), attested in 1457.</watermark>
+    *   <measure type="chainline">Chain distance 35mm.</measure>
+    * </support>
+    * We extract the watermarks but we filter out the measure for now
+    */
+  private def parseSupport(supportDesc: Node) = {
+    val support = (supportDesc \ "support")
+    val supportStr = if (support.exists(_.child.size > 1)) {
+      val watermarkStr = (support \ "watermark").text.trim
+      val supportLabel = support
+        .flatMap(_.child)
+        .collect {
+          case node if node.label != "watermark" && node.label != "measure" =>
+            node.text.trim
+        }
+        .mkString(" ")
+        .trim
+      List(
+        supportLabel,
+        if (watermarkStr.nonEmpty) s"Watermarks: $watermarkStr" else "")
+        .filterNot(_.isEmpty)
+        .mkString("; ")
```

Comment:
Intermediate variable just to break it up slightly:

```suggestion
      val parts = List(supportLabel, if (watermarkStr.nonEmpty) s"Watermarks: $watermarkStr" else "")
      
      parts
        .filterNot(_.isEmpty)
        .mkString("; ")
```